test/acceptance/ignoreCase.robot

```diff
@@ -0,0 +1,19 @@
+*** Settings ***
+Library           SeleniumLibrary
+Test Teardown     Close All Browsers
+
+*** Variables ***
+${ISSUE_URL}      https://github.com/robotframework/SeleniumLibrary/issues/849
```

Comment:
Using links to external resources are not allowed. The acceptance test runner starts small http server. which must be used to serve the required html files. Please take an example from the ./test/acceptance/keywords/content_assertions.robot suite. 

The test logic is OK,